components/script/dom/htmlmediaelement.rs

```diff
@@ -324,6 +324,11 @@ impl HTMLMediaElement {
             playground::player::PlayerEvent::EndOfStream => {
                 println!("EOS!");
             }
+            playground::player::PlayerEvent::Error => {
+                println!("Error!");
+                self.error.set(Some(&*MediaError::new(&*window_from_node(&*self), MEDIA_ERR_DECODE)));
```

Comment:
Later on we might need to add a parameter to the enum so we can report a different error here. Not all errors come from the decoder.